src/screens/FormField.js

```diff
@@ -558,6 +558,26 @@ const FormFieldPage = () => (
           </PropertyValue>
         </Property>
 
+        <Property name="formField.kind.label">
+          <Description>
+            Any props of Text that will be applied on the FormField `kind`
+            label.
+          </Description>
+          <PropertyValue type="object">
+            <Example>
+              {`
+              {
+                size: 'large',
+                color: 'black',
+                margin: {
+                  vertical: "xsmall",
+                }
+              }
+            `}
```

Comment:
```suggestion
              {`
{
  size: 'large',
  color: 'black',
  margin: {
    vertical: "xsmall",
  }
}
            `}
```
The extra spacing here will show up in the example on the site